tests/__compound-components__/Filters/Facets.tsx

```diff
@@ -1,8 +1,9 @@
 import { Filters } from '../../../src/components';
 
-export function Facets({ searchOnChange = true }: { searchOnChange?: boolean }): JSX.Element {
+export function Facets(args: Filters.FacetsProps): JSX.Element {
+  const searchOnChange = args.searchOnChange === false ? false : true;
```

Comment:
Do we need to merge this with `args`?

Reply:
It's not necessary, because `searchOnChange` is true by default in the `Facets` component. But, I updated to merge it in for the sake of clarity